depmapomics/dm_omics.py

```diff
@@ -161,9 +161,17 @@ async def expressionPostProcessing(
     if run_stranded:
         pr_files_stranded = dict()
         tpm_mat = files_stranded["proteincoding_genes_tpm_stranded"]
-        pr_files_stranded["proteincoding_genes_tpm_profile_stranded"] = tpm_mat[
-            tpm_mat.index.isin(set(renaming_dict.keys()))
-        ].rename(index=renaming_dict)
+        all_genes_mat = files_stranded["genes_tpm_stranded"]
+        transcripts_mat = files_stranded["transcripts_tpm_stranded"]
+        pr_files_stranded["proteincoding_genes_tpm_profile_stranded"] = tpm_mat[tpm_mat.index.isin(set(renaming_dict.keys()))].rename(
```

Comment:
`df[df.index.isin(set([1, 2, 10]))]` is equivalent to `df.loc[[1, 2, 10]]` so I think these could all be written as just, e.g., `tpm_mat.loc[renaming_dict.keys()]`

Reply:
Yeah I have the tendency to abuse `index.isin()`, but in this case `renaming_dict` has more elements than the indices in `df`. Specifically, `renaming_dict` has the SeqID->PRID mapping for all sequencing types, but this df only has RNA. Here I think using `df.loc` would error

Reply:
Makes sense